jwala-services/src/main/java/com/cerner/jwala/service/binarydistribution/impl/BinaryDistributionServiceImpl.java

```diff
@@ -199,21 +201,31 @@ public boolean remoteFileCheck(final String hostname, final String remoteFilePat
         return result;
     }
 
+    /**
+     * Distrubute unzip.exe for windows and unzip.sh for other OS
+     * @param hostname host name
+     */
     @Override
     public void distributeUnzip(String hostname) {
         LOGGER.info("Start deploy unzip for {}", hostname);
         final String jwalaScriptsPath = ApplicationProperties.getRequired(PropertyKeys.REMOTE_SCRIPT_DIR);
         if (!remoteFileCheck(hostname, jwalaScriptsPath)) {
             remoteCreateDirectory(hostname, jwalaScriptsPath);
         }
-
         if (!remoteFileCheck(hostname, jwalaScriptsPath + "/" + UNZIPEXE)) {
             final String unzipFile = ApplicationProperties.get(PropertyKeys.LOCAL_JWALA_BINARY_DIR) + "/" + UNZIPEXE;
             LOGGER.info("SCP {} " + unzipFile);
             remoteSecureCopyFile(hostname, unzipFile,  jwalaScriptsPath + "/" + UNZIPEXE);

```

Comment:
No the our windows unzip.exe will not work on linux. Linux comes with unzip sh the script will work. Good catch, I didn't realize we took it out of the bundle(which is good), I will add a check if file exists then scp, so I will not have to check host OS